qemu/tests/cfg/watchdog.cfg

```diff
@@ -22,6 +22,12 @@
             only s390x
             watchdog_device_type = diag288
             module_check_cmd = "lsmod | grep diag288"
+        -itco:
+            required_qemu = [8.0.0,)
+            only q35
+            watchdog_device_type = itco
+            module_check_cmd = "lsmod | grep iTCO_wdt"
```

Comment:
The module `iTCO_mdt` is not a built-in module as `CONFIG_ITCO_WDT=m`. 
Suggest load it all the time

Reply:
@yanan-fu ,Linux guests automatically detect the TCO watchdog ahd load the 'iTCO_wdt' kmod to enable it. not need to re-load it.thanks

Reply:
But you still add the step to check if the module be loaded or not ?

Leave the default behaviour,  we are sure it is not a build-in module, can be load/unload.  then make the test env meet the test requirements is needed.

Reply:
I think that iTCO_mdt is  a built-in module, only need to check that the module is automatically loaded successfully, and  not need to re-load the module

Reply:
It is not build-in, as i commented in the first comment   `CONFIG_ITCO_WDT=m.`